common/kms_wallet_config.go

```diff
@@ -0,0 +1,42 @@
+package common
+
+import (
+	"github.com/urfave/cli"
+)
+
+type KMSKeyConfig struct {
+	KeyID   string
+	Region  string
+	Disable bool
+}
+
+func KMSWalletCLIFlags(envPrefix string, flagPrefix string) []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:     PrefixFlag(flagPrefix, "kms-key-id"),
+			Usage:    "KMS key ID that stores the private key",
+			Required: false,
+			EnvVar:   PrefixEnvVar(envPrefix, "KMS_KEY_ID"),
```

Comment:
Should we use a [KMS alias](https://docs.aws.amazon.com/kms/latest/developerguide/kms-alias.html) rather than the key id? Think it would help when rotating the key since we wouldn't need to do a deployment.

Reply:
Sounds good. This has to happen in sdk first, so I'll merge this and implement that as next iteration

Reply:
Apparently it just works: https://docs.aws.amazon.com/kms/latest/APIReference/API_Sign.html#API_Sign_RequestSyntax